### PR TITLE
feat: Add appLocale capability

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: WebDriverAgentMac/WebDriverAgentLib/Commands/FBSessionCommands.m [hunk 1]
```diff
@@ -234,4 +241,30 @@ + (NSDictionary *)currentCapabilities
...
+  if (nil != appLocale[@"locale"]) {
+    [result addObject:@"-AppleLocale"];
+    [result addObject:[NSString stringWithFormat:@"%@", appLocale[@"locale"]]];
+  }
+  if (nil != appLocale[@"useMetricUnits"]) {
+    [result addObject:@"-AppleMetricUnits"];
+    [result addObject:[NSString stringWithFormat:@"<%@/>", appLocale[@"useMetricUnits"] ? @"true" : @"false"]];
+  }
+  if (nil != appLocale[@"measurementUnits"]) {
+    [result addObject:@"-AppleMeasurementUnits"];
```
Comment (KazuCocoa) [inline WebDriverAgentMac/WebDriverAgentLib/Commands/FBSessionCommands.m:261]: May `AppleTemperatureUnit` also work?
https://github.com/apple/swift-corelibs-foundation/blob/7f62c8c52aeed9a86b503fe7b662c48eda5d0f8a/CoreFoundation/Locale.subproj/CFLocale.c#L1593C48-L1593C68

**Review by KazuCocoa — APPROVED** [review]
Nice, didn't know of measurements

File: WebDriverAgentMac/WebDriverAgentLib/Commands/FBSessionCommands.m (hunk 1 above)
Comment (mykola-mokhnach) [reply WebDriverAgentMac/WebDriverAgentLib/Commands/FBSessionCommands.m:261]: maybe, not sure how to test that though...

Comment (mykola-mokhnach) [reply WebDriverAgentMac/WebDriverAgentLib/Commands/FBSessionCommands.m:261]: feel free to add it if you know

**Pull request merged**